.changeset/shy-impalas-float.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/design-system-components": patch
+---
+
+`SideNav` - Fixed visible scrollbar in collapsed SideNav when scroll bar is set to be always visible
```

Comment:
👏

Reply:
Sure thing – added in [4a4a3e3](https://github.com/hashicorp/design-system/pull/1786/commits/4a4a3e3f97cc4d8a2004512ded250cb63c48230f)